app/controllers/claims/support/claim_windows_controller.rb

```diff
@@ -69,4 +55,12 @@ def set_claim_window
   def authorize_claim_window
     authorize @claim_window || Claims::ClaimWindow
   end
+
+  def claim_window_form
+    @claim_window_form ||= if params[:claims_claim_window_form]
+                             Claims::ClaimWindowForm.new(id: params[:id], **claim_window_form_params)
+                           else
+                             Claims::ClaimWindowForm.new(id: params[:id])
+                           end
```

Comment:
Since you never want to raise a [`ActionController::ParameterMissing`](https://api.rubyonrails.org/classes/ActionController/ParameterMissing.html) error, it would be more succinct to use `params.fetch` rather than `params.require`.

```diff
def claim_window_form_params
-  params.require(:claims_claim_window_form).permit(:starts_on, :ends_on, :academic_year_id)
+  params.fetch(:claims_claim_window_form, {}).permit(:starts_on, :ends_on, :academic_year_id)
end
```

This way, the `claim_window_form_params` method will always return a params object that can be safely expanded using the double splat operator.

```suggestion
    @claim_window_form ||= Claims::ClaimWindowForm.new(id: params[:id], **claim_window_form_params)
```